packages/app/src/components/MinistereTravail/SearchField.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx jsx */
+import { ChangeEventHandler, FC } from "react";
+import { css, jsx } from "@emotion/core";
+
+interface FieldProps {
+  name: string;
+  onChange?: ChangeEventHandler;
+  placeholder?: string;
+  value?: string;
+}
+
+const SearchField: FC<FieldProps> = ({
+  name,
+  onChange,
+  placeholder,
+  value
+}) => (
+  <input
+    name={name}
+    value={value}
+    onChange={onChange}
+    placeholder={placeholder}
+    css={styles.fieldMinistere}
+    type="text"
+  />
+);
+
+const styles = {
+  fieldMinistere: css({
+    width: "200px",
+    height: "40px",
+    paddingLeft: "10px",
+    border: "unset",
+    // Remove the clear field cross from IE.
```

Comment:
On dirait que c'est aussi possible sur Chrome https://blog.maximerouiller.com/post/remove-the-x-from-internet-explorer-and-chrome-input-type-search/ (de mémoire il n'y en a pas sur Firefox ?)

Reply:
J'en vois pas sur chrome :(

Reply:
my bad, je pensais qu'il y en avait par défaut sur les searchbox sur chrome, mais je confirme que je n'en vois pas